cypress-action/cypress/e2e/e2e/loadDesign.js

```diff
@@ -64,12 +64,19 @@ const captureSnapshot = ({ window, designId, theme }) => {
 
 const removeWidgets = () => {
     const classes = ["MuiToolbar-root", "MuiDrawer-paperAnchorDockedLeft", "MuiSpeedDial-root"] ;
```

Comment:
@jerensl  lets replace these with dedicated ids so we dont accidenlty remove important elements .